docs/antrea-network-policy.md

```diff
@@ -272,6 +274,22 @@ single port, to the 10.0.10.0/24 subnet specified by the `ipBlock` field.
 **Note**: The order in which the egress rules are set matter, i.e. rules will
 be enforced in the order in which they are written.
 
+**enableLogging**: A ClusterNetworkPolicy ingress or egress rule can be
+audited by enabling its logging field. When `enableLogging` field is set to
+true, the first packet of any connection that matches this rule will be logged
+to a separate file (`/var/log/antrea/networkpolicy/np.log`). These log files
+can then be retrieved for further analysis. By default, rules are not logged.
+The example policy logs all traffic that matches the "DropToThirdParty" egress
+rule, while the rule "AllowFromFrontend" is not logged.
+The rules are logged in the following format:
+
+```
+    <yyyy/mm/dd> <time> <ovs-table-name> <antrea-native-policy-reference> <action> <openflow-priority> SRC: <source-ip> DEST: <destination-ip> <packet-length> <protocol>
+
+    Example:
+    2020/11/02 22:21:21.148395 AntreaPolicyAppTierIngressRule AntreaNetworkPolicy:default/test-anp Allow 61800 SRC: 10.0.0.4 DEST: 10.0.0.5 60 TCP
```

Comment:
I know this is just a documentation PR, but the inclusion of `SRC:` and `DEST:` looks really weird...
We don't provide a key for any other field

Reply:
maybe we can follow up on the log format.. i remember @jianjuns had some suggestions around this, even suggested using binary format to log.. but maybe we can follow it up on an issue

Reply:
Just a reminder here that if the log format changes, we need to update the regex expression for logging in the e2e testcase in `test/e2e/antreapolicy_test.go`

Reply:
yes, I think we should have a small PR before the v0.11 release to remove `SRC:` and `DEST:`, unless someone is strongly attached to them

either we should have structured logs with KVs, or omit all field names (no reason to have `SRC` but not `OF_PRIORITY` for example).